apps/prairielearn/src/pages/instructorInstanceAdminAccess/instructorInstanceAdminAccess.js

```diff
@@ -1,23 +1,31 @@
 // @ts-check
-const ERR = require('async-stacktrace');
+const asyncHandler = require('express-async-handler');
 import * as express from 'express';
+import { z } from 'zod';
 
 import * as sqldb from '@prairielearn/postgres';
 
 const router = express.Router();
 const sql = sqldb.loadSqlEquiv(__filename);
 
-router.get('/', function (req, res, next) {
-  var params = {
-    course_instance_id: res.locals.course_instance.id,
-  };
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    res.locals.access_rules = await sqldb.queryRows(
+      sql.course_instance_access_rules,
+      {
+        course_instance_id: res.locals.course_instance.id,
+      },
```

Comment:
This is really a nit, but it bugs me that prettier needs a push to make this work.

```suggestion
      { course_instance_id: res.locals.course_instance.id },
```